### DISCUSSION TITLE
Elaborate on ChangeLog Versions 42

### DISCUSSION THREAD

**Discussion by dadams39** [microsoftgraph/msgraph-sdk-go#301] [General]
ChangeLog describes the version change

```
BREAKING: removed overloads to get request information.
```

Is there any more information on what was changed and what was broken?

**Comment by baywet** [comment]
CreateGetRequestInformation use to have - overloads (With suffix) to mimic optional parameters. Now those are gone and all the parameters (or nil) must be passed to the remaining method.
